image_examples/README.md

```diff
@@ -28,3 +28,15 @@ TIMEPIX_SU_512-stdgoni_0001.img
 TIMEPIX_SU_516-stdgoni_0001.img
 - Courtesy of Viktor Bengtsson (2019)
 - For testing FormatSMVTimePix_SU_516x516
+
+Dectris_Nexus_master.h5\
+Dectris_Nexus_data_000001.h5
```

Comment:
```suggestion
Dectris_Eiger_master.h5\
Dectris_Eiger_data_000001.h5
```

Reply:
do we need capital letters at all?

Reply:
Oops, good spot, thanks.  Thought I'd done that.  But yeah all-lowercase probably better.